src/octopeer-github/content/ContentController.ts

```diff
@@ -94,17 +94,13 @@ class ContentController {
      * Hook the different rawdata trackers to DOM.
      * @param database   the database that should be used when logging.
      */
+    // tslint:disable:no-unused-variable
     private hookSyntacticToDOM(database: DatabaseAdaptable) {
-        let windowResolutionTracker: WindowResolutionTracker;
-        let keystrokeTracker: KeystrokeTracker;
-        let mouseClickTracker: MouseClickTracker;
-        let mouseScrollTracker: MouseScrollTracker;
-        let mousePositionTracker: MousePositionTracker;
-
-        windowResolutionTracker = new WindowResolutionTracker(database);
-        keystrokeTracker = new KeystrokeTracker(database);
-        mouseClickTracker = new MouseClickTracker(database);
-        mouseScrollTracker = new MouseScrollTracker(database);
-        mousePositionTracker = new MousePositionTracker(database);
+        let windowResolutionTracker = new WindowResolutionTracker(database);
+        let keystrokeTracker = new KeystrokeTracker(database);
+        let mouseClickTracker = new MouseClickTracker(database);
+        let mouseScrollTracker = new MouseScrollTracker(database);
+        let mousePositionTracker = new MousePositionTracker(database);
     }
+    // tslint:enable:no-unused-variable
```

Comment:
Can you move the tslint disable/enable inside the method? I think this is cleaner, since the method signature has nothing to do with it.